src/main/java/de/tum/in/www1/artemis/config/Constants.java

```diff
@@ -7,6 +7,9 @@
  */
 public final class Constants {
 
+    // ToDo Change to 24h before merging into production
+    public static int COMPLAINT_LOCK_DURATION_IN_MINUTES = 5;
```

Comment:
Don't forget to change the duration before merging

Reply:
Hopefully will not forget it :)

Reply:
I think this should be done now. When we merge it into production nobody will think of this TODO. 
This would mean, that before merging into production a new commit would be needed, which then would be off with the development branch.